### DISCUSSION TITLE
[General] Cyrillic support in GLPI agent

### DISCUSSION THREAD

**Discussion by tuxzer92** [glpi-project/glpi-agent#596] [General]
### Your question

Hi all

GLPI Server on Debian 12.4
GLPI version: GLPI 10.0.11
glpi-agent version: 1.7.1-1
computers for inventory on Win10

When I go to the toolbox page and enter credentials using a login in Russian, I now cannot get to the page for setting up IP address ranges:
`ERR_EMPTY_RESPONSE`

There is an error in the GLPI agent log:
`
[debug] [http server] GET request /toolbox/ip_range from client 192.168.31.79
[Fri Feb  2 12:42:37 2024][error] HTTP::Message content must be bytes at /usr/share/glpi-agent/lib/GLPI/Agent/HTTP/Server/ToolBox.pm line 853.
`
If I enter the credentials in English, then everything works. How can I make it possible to use Russian letters in the GLPI agent?

**Comment by g-bougard** [comment]
Hi @tuxzer92 

I just pushed some code that could fix your issue.

So can you test with the next [nightly build](https://nightly.glpi-project.org/glpi-agent/) ?
